docker-compose.yml

```diff
@@ -32,3 +33,13 @@ services:
       - redis:/data
     ports:
       - "6379:6379"
+
+  search:
+    image: getmeili/meilisearch:v1.9.0
+    restart: always
+    environment:
+        MEILI_MASTER_KEY: topsecret
+    volumes:
+     - search:/data.ms
+    ports:
+      - '7700:7700'
```

Comment:
@omBratteng I added this to temp have local meilisearch instance, can this be moved to local adhoc somehow? Do we have support for it, it works as is just want to get a feeling since it will allow us to play with search locally? cc: @rebelchris

Reply:
Would be a super addition to our local instance, really helps with any search debugging.
(And nice pro to have the local meili interface to play with)